test/e2e/upgrade_management_cluster_test.go

```diff
@@ -266,4 +266,24 @@ func preCleanupManagementCluster(clusterProxy framework.ClusterProxy) {
 		}
 	})
 	reInstallIronic()
+
+	//#nosec G204 -- We need to pass in the file name here.
+	cmd := exec.Command("bash", "-c", "kubectl apply -f bmhosts_crs.yaml  -n metal3")
+	cmd.Dir = workDir
+	output, err := cmd.CombinedOutput()
+	Logf("Applying bmh to meta3 namespace : \n %v", string(output))
```

Comment:
where are we removing the namespace as mentioned in the description?

Reply:
The upgrade management test will remove the namespace generated randomly it doesn't use meta3 ns this namespace should be created by dev-env and stays always there